### PR TITLE
Add Code Thaw deadline to 1.19 release cycle

### PR DESCRIPTION
**Pull request by saschagrunert**
#### What type of PR is this:
/kind documentation

#### What this PR does / why we need it:
We seem to have missed this point so we now add it one week prior to the
actual release.

#### Which issue(s) this PR fixes:

None
#### Special notes for your reviewer:
/assign @onlydole @justaugustus

### CODE REVIEW

File: releases/release-1.19/README.md [hunk 1]
```diff
@@ -76,6 +76,7 @@ The 1.19 release cycle is proposed as follows:
 | **Cherry Pick Deadline** (EOD PST) | Branch Manager | Thu, August 6 | week 17 | |
 | **Test Freeze** (EOD PST) | Branch Manager | Thu, August 6 | week 17 | |
 | **KubeCon CloudNativeCon Europe  -  Break** | Community | Mon, August 10 | week 18 | |
+| **[Code Thaw]** (EOD PST) | Branch Manager | Thu, August 18 | week 19 | |
```
Comment (saschagrunert) [inline releases/release-1.19/README.md:79]: I't now directly in the KubeCon break, do we wanna move it before the break?

Comment (markyjackson-taulia) [reply releases/release-1.19/README.md:79]: I say yes to moving before the break

**Comment by justaugustus** [comment]
/hold it was purposely excluded
I'll look at this later.

File: releases/release-1.19/README.md [hunk 2]
```diff
@@ -75,6 +75,7 @@ The 1.19 release cycle is proposed as follows:
 | Burndown Meetings daily | Lead | Mon, August 3 | week 16 | |
 | **Cherry Pick Deadline** (EOD PST) | Branch Manager | Thu, August 6 | week 17 | |
 | **Test Freeze** (EOD PST) | Branch Manager | Thu, August 6 | week 17 | |
+| **[Code Thaw]** (EOD PST) | Branch Manager | Mon, August 10 | week 18 | |
```
Comment (saschagrunert) [inline releases/release-1.19/README.md:78]: Moved it there to not hit the kubecon break.

File: releases/release-1.19/README.md (hunk 1 above)
Comment (saschagrunert) [reply releases/release-1.19/README.md]: Sounds good!

**Comment by onlydole** [comment]
Thank you @saschagrunert and @markyjackson-taulia - this looks good to me, barring any feedback from the SIG chairs
/lgtm

**Comment by saschagrunert** [comment]
Just saw that we announced code thaw for July 21th. So I think the docs should reflect that.

**Comment by saschagrunert** [comment]
Rebased to resolve the conflicts.

**Pull request merged**